### PR TITLE
42 modal redesign

### PR DESCRIPTION
**Pull request by EliasPinno**
From the ticket description:

- [x] Modal heading should be changed from "CHIKV Estimate Study", "ZIKV Estimate Study", etc. to "CHIKV Estimate", "ZIKV Estimate".
- [x] The url to the study with a hyperlink to that url should be replaced with the name of the study which is still hyperlinked to the url of the study.
- [x] The gray ribbon on the modal where seroprevalence is should match the colour of the study, Ex. green for West Nile Virus, light blue for Chikungaya Virus, etc.
- [x] The gray ribbon should also include "Arbovirus: Zika Virus" or "Arbovirus: West Nile Virus", etc.
- [x] The sampling date should be in it's own column and not bunched together with the seroprevalence.
- [x] LatLng row should be removed
- [x] "Pathogen: ZIKV" or "Pathogen: WNV" at the bottom of the modal should be removed.

I didn't implement the URL being renamed to the study name: this is because I can't see on the 'record' object the study name being extracted. I think the backend needs to be altered to return as part of the response the study name.

Modal-Redesign

### CODE REVIEW

File: app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx [hunk 1]
```diff
@@ -26,6 +26,25 @@ function pathogenTag(pathogen: string) {
...
+          return "Yellow Fever";
+        case "WNV":
+          return "West Nile Virus";
+        case "MAYV":
+          return "Mayaro Virus";
+        default:
+          return "Unknown";
+      }
+}
+
```
Comment (EliasPinno) [inline app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx:47]: Should this be defined elsewhere? Is this a good approach for what I am doing here?

File: app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx [hunk 2]
```diff
@@ -53,36 +71,27 @@ export default function ArboStudyPopup(record: any) {
...
+                    {"Seroprevalence"}: <b> {`${(record.seroprevalence * 100).toFixed(1)}% (Arbovirus: ${pathogenFullString(record.pathogen)})`}</b>
                 </div>
             </div>
             {/*Content section*/}
             <div className={"py-2 px-4 max-h-[250px] overflow-auto"}>
+                {row("Date", `${TranslateDate(record.sample_start_date)} to ${TranslateDate(record.sample_end_date)}`)}
                 {row("Inclusion Criteria", record.inclusion_criteria ? record.inclusion_criteria : "Not Reported")}
                 {row("Location", getGeography(record.city, record.state, record.country))}
                 {row("Sample Size", record.sample_size?.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ","))}
                 {row("Antibody Target",  record.antibodies.join(', '))}
```
Comment (EliasPinno) [inline app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx:92]: Should I color Antibody target? It was not requested in the ticket but we could color it like how we do in the table.

File: app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx [hunk 3]
```diff
@@ -26,6 +26,25 @@ function pathogenTag(pathogen: string) {
     )
 }
 
+function pathogenFullString(pathogen: string) {
```
Comment (SeanKennyNF) [inline app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx:29]: > The url to the study with a hyperlink to that url should be replaced with the name of the study which is still hyperlinked to the url of the study.

Commenting on random line for thread but I'd imagine we can use the `source_sheet_name` as the name of the study for the purposes of the modal. Source sheet names look like this: "A cross-sectional study of acute dengue infection in paediatric clinics in Cameroon" or "Serological and spatial analysis of alphavirus and flavivirus prevalence and risk factors in a rural community in western Kenya"

cc @HimanshuRanka @harrietware @gurickkooner who can correct me if I'm wrong here

File: app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx (hunk 1 above)
Comment (SeanKennyNF) [reply app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx:47]: No problems with this approach imo. We could maybe move this to a helper seeing as we'll need it in other spots too but totally something we can hit later :+1:

File: app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx [hunk 4]
```diff
@@ -53,36 +71,27 @@ export default function ArboStudyPopup(record: any) {
...
-                        </>)
-                    }
+            <div className={`flex justify-between bg-${record.pathogen.toLowerCase()} w-full py-2 px-4`}>
+                <div className={"font-semibold"}>
+                    {"Seroprevalence"}: <b> {`${(record.seroprevalence * 100).toFixed(1)}% (Arbovirus: ${pathogenFullString(record.pathogen)})`}</b>
                 </div>
             </div>
             {/*Content section*/}
             <div className={"py-2 px-4 max-h-[250px] overflow-auto"}>
+                {row("Date", `${TranslateDate(record.sample_start_date)} to ${TranslateDate(record.sample_end_date)}`)}
```
Comment (SeanKennyNF) [inline app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx:88]:
```suggestion
                {row("Sampling Date Range", `${TranslateDate(record.sample_start_date)} to ${TranslateDate(record.sample_end_date)}`)}
```
If it's to your taste hahaha

File: app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx (hunk 2 above)
Comment (SeanKennyNF) [reply app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx:92]: I think the modal has pretty minimal color and looks okay the way it is but if that's something you'd like to do I'm down.

**Review by SeanKennyNF — APPROVED** [review]

**Review by HimanshuRanka — CHANGES_REQUESTED** [review]
One teeny change that is necessary here to change date -> sampling date range as Sean commented but looks really great otherwise! Can also add colors to the antibody type as you mentioned

File: app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx (hunk 2 above)
Comment (EliasPinno) [reply app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx]: It is colored and spaced correctly!

File: app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx (hunk 3 above)
Comment (EliasPinno) [reply app/pathogen/arbovirus/dashboard/ArboStudyPopup.tsx:29]: We solved this in the meeting!

**Comment by EliasPinno** [comment]
@HimanshuRanka Just approve and I think this is good to merge now.

**Comment by HimanshuRanka** [comment]
@EliasPinno  I cannot see the sampling date range on the modal?

**Comment by EliasPinno** [comment]
Export-Data-To-CSV
Fixed both, it was because I was resolving a merge conflict at 1am...

**Pull request merged**